setup.py

```diff
@@ -40,8 +40,7 @@ def get_version():
     test_suite='tests',
     install_requires=[
         'GitPython>=2.1.8,<3',
-        'pylint>=1.7.1,<1.8',
-        'astroid>=1.5,<1.6',
```

Comment:
Can we still restrict this to avoid 1.6.0?

Reply:
yes totally I've abandoned this PR while back but i'm gonna go back on this and will make sure to exclude `1.6.0`